asv_bench/benchmarks/io/csv.py

```diff
@@ -62,6 +55,10 @@ class TimeReadCsvSkiprows(BaseReadCsv):
         ],
     ]
 
+    def setup_cache(self, test_filename="io_test_file"):
+        test_filenames = prepare_io_data(test_filename, "str_int", self.shapes)
```

Comment:
Just store `"str_int"` as a class attribute, implement this `setup_cache()` once in parent and use the power of OOP to _not_ copy-paste this everywhere.

That said, if you use the argument to `get_benchmark_shapes()` that is always equal to the class name, why don't you do something like `prepare_io_data(test_filename, self.data_type, get_benchmark_shapes(self.__class__.__name__))` and _not_ store the `shape` attribute at all?